samples/msgext-ai-doc-compliance-checker/README.md

```diff
@@ -0,0 +1,130 @@
+---
+page_type: sample
+description: This sample demonstrates integrating Azure Open AI into a Teams message extension to check a contract proposal's compliance with a policy checklist. It works with Teams Toolkit for Visual Studio Code and JavaScript and can be used in Copilot for Microsoft 365.
+products:
+- office-teams
+- copilot-m365
+- azure open-ai
+languages:
+- javascript
+---
```

Comment:
This portion is missing here:

 extensions:
 contentType: samples
 createdDate: "07/15/2024 01:38:25 PM"
urlFragment: officedev-microsoft-teams-samples-bot-ai-virtual-assistant-nodejs
---

Reply:
Refer this PR: https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/1335/files#diff-820624b3d7f7e4be48d0fc48df9ab543ee07089829a4bf35bd2e808c12637bb9

Reply:
Fixed